### ISSUE TITLE
some obligees seems to be always changed, even when not so

### ISSUE THREAD

**Issue by mmmaly** [martinmacko47/chcemvediet#163]
after importing a new listObligees.xlsx, the following obligees change, even when no change  was made in xlsx for their entries. I do not know, whether Excel makes any modification internally.

Changed <Obligee: [5639] Obec Lovce>
Changed <Obligee: [3450] Materská škola, Uhorské>
Changed <Obligee: [7323] Praktická škola, Vrbové>
Changed <Obligee: [12021] Špeciálna materská škola, Vrbové>
Changed <Obligee: [12162] Špeciálna základná škola, Vrbové>
Changed <Obligee: [14256] Základná škola, Uhorské>

**Comment by martinmacko47** [comment]
If you reimport the new xlsx twice in a row, will they report changed again?

**Comment by mmmaly** [comment]
yes.

```
Importing: private/listObligees-20161102.xlsx
Importing Region...
Imported Region: 0 created, 0 changed, 9 unchanged and 0 marked for deletion
Importing District...
Imported District: 0 created, 0 changed, 80 unchanged and 0 marked for deletion
Importing Municipality...
Imported Municipality: 0 created, 0 changed, 2928 unchanged and 0 marked for deletion
Importing Neighbourhood...
Imported Neighbourhood: 0 created, 0 changed, 7034 unchanged and 0 marked for deletion
Importing ObligeeTag...
Imported ObligeeTag: 0 created, 0 changed, 1 unchanged and 0 marked for deletion
Importing ObligeeGroup...
Imported ObligeeGroup: 0 created, 0 changed, 67 unchanged and 0 marked for deletion
Importing Obligee...
Changed <Obligee: [5639] Obec Lovce>
Changed <Obligee: [3450] Materská škola, Uhorské>
Changed <Obligee: [7323] Praktická škola, Vrbové>
Changed <Obligee: [12021] Špeciálna materská škola, Vrbové>
Changed <Obligee: [12162] Špeciálna základná škola, Vrbové>
Changed <Obligee: [14256] Základná škola, Uhorské>
Imported Obligee: 0 created, 6 changed, 9741 unchanged and 0 marked for deletion
Importing ObligeeAlias...
Imported ObligeeAlias: 0 created, 0 changed, 1 unchanged and 0 marked for deletion
Done.
```

**Comment by martinmacko47** [comment]
Caused by longitude/latitude float precision errors. Fixed with #164 by considering floats less than 10<sup>-7</sup> off equal.

**Comment by martinmacko47** [comment]
Fixed in #164
